### PR TITLE
Remove unhealhty db unless it is master type

### PR DESCRIPTION
**Pull request by dangra**

### CODE REVIEW

File: cmd/sentinel/cmd/sentinel.go [hunk 1]
```diff
@@ -974,7 +974,7 @@ func (s *Sentinel) updateCluster(cd *cluster.ClusterData, pis cluster.ProxiesInf
 
 				// get db associated to the keeper
 				db := cd.FindDB(k)
-				if db != nil {
+				if db != nil && s.dbType(newcd, db.UID) != dbTypeMaster {
```
Comment (davissp14) [inline cmd/sentinel/cmd/sentinel.go:977]: I think we will want to ensure that both the keeper and db remain intact in the event it's master.

**Review by davissp14 — CHANGES_REQUESTED** [review]
I think we will want to keep the db and keeper intact in the event it's master.

**Comment by dangra** [comment]
Oops, that was an oversight. Addressed by 591d6ae5473032504c5d52b8738c3d332244f00e

**Review by davissp14 — APPROVED** [review]
Looks good!

**Pull request merged**